packages/create-svelte/shared/+eslint+prettier/package.json

```diff
@@ -3,7 +3,7 @@
 		"eslint-config-prettier": "^8.5.0"
```

Comment:
Can't `eslint-config-prettier` be bumped to v9? It does not depend on Prettier v3, but it is the only outdated package as of now. [Changelog](https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md)

Reply:
yes. thanks for the suggestion. PR here: https://github.com/sveltejs/kit/pull/11037